Triangulation_2/include/CGAL/Regular_triangulation_euclidean_traits_2.h

```diff
@@ -24,6 +24,11 @@
 
 #include <CGAL/license/Triangulation_2.h>
 
+#ifdef CGAL_NO_DEPRECATED_CODE
```

Comment:
I think this should be `#ifndef`, instead of `#ifdef`
And, should it really make an error? A warning would be enough, maybe...?